azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/configuration/CallCompositeConfiguration.kt

```diff
@@ -3,12 +3,21 @@
 
 package com.azure.android.communication.ui.calling.configuration
 
+import android.content.Context
+import com.azure.android.communication.common.CommunicationTokenCredential
 import com.azure.android.communication.ui.calling.configuration.events.CallCompositeEventsHandler
+import com.azure.android.communication.ui.calling.models.CallCompositeCallScreenOptions
 import com.azure.android.communication.ui.calling.models.CallCompositeLocalOptions
 import com.azure.android.communication.ui.calling.models.CallCompositeLocalizationOptions
 import com.azure.android.communication.ui.calling.models.CallCompositeSupportedScreenOrientation
+import com.azure.android.communication.ui.calling.models.CallCompositeTelecomOptions
 
 internal class CallCompositeConfiguration {
+    var callScreenOptions: CallCompositeCallScreenOptions? = null
+    var context: Context? = null
```

Comment:
We need a context in configuration? Application/Activity?

Does not seem right, is there another alternative. Context itself is not a configuration field.

Reply:
Sounds good Adam, we can remote it from here and keep in builder until call agent is crated.